tests/models/hydrology/test_above_ground.py

```diff
@@ -60,21 +60,25 @@ def test_calculate_soil_evaporation(dens_air, latvap):
     )
 
 
-def test_find_lowest_neighbour(dummy_climate_data):
+def test_find_lowest_neighbour(fixture_core_components, dummy_climate_data):
     """Test finding lowest neighbours."""
 
-    from math import sqrt
+    # FIXME: At the moment this is being tested on a 2x2 grid with these elevations and
```

Comment:
Sorry, I don't understand this commend, what would you suggest to change?

Reply:
Yeah - bit obscure 😄.

This doesn't need fixing now. I'm not convinced that the function is doing what we want right now - but might be wrong. So at some point (possibly when this gets ported to `Grid`?) we should review these test values and the implementation.